qubes/storage/__init__.py

```diff
@@ -339,6 +381,43 @@ def size(self, size):
         # pylint: disable=attribute-defined-outside-init
         self._size = int(size)
 
+    # pylint: disable=no-self-use
+    def volatile_volume_path(self, qube_name, device_name):
```

Comment:
Maybe rather name it with `encrypted` or `ephemeral` instead of `volatile`? Otherwise it can be easily confused with the underlying `volatile` volume.